packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -298,6 +298,44 @@ skipInNode(describe)('plugin-meetings', () => {
         assert.exists(alice.meeting.members.selfId, 'selfId not present');
       });
 
+      it('alice resolution is 1080p, bob receives 1080p', () => {
+        testUtils.delayedPromise(alice.meeting.mediaProperties.setLocalQualityLevel('1080p'));
```

Comment:
I don't see how these tests can work at all, there are multiple issues with them:
1. All calls in all these tests need to be wrapped with `return Promise.all()` otherwise the test runner will think that the tests are completed before the code that is executed in them actually completes.
2. The return value of `testUtils.waitForEvents()` is ignored, so the test doesn't actually check if these events fired and also the checks inside the `then()` after waitForEvents are not actually executed.
3. The tests call meeting.mediaProperties.setLocalQualityLevel() which does nothing except for storing the value - see here: https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/plugin-meetings/src/media/properties.js#L62 - they should instead call `meeting.setLocalVideoQuality()`

Reply:
Yes, I realised that. Updated the tests.